styles.css

```diff
@@ -307,6 +282,7 @@ h4 {
   height: 48px;
   padding: 12px;
   text-align: center;
+  margin-top: -37px;
 }
 
 .project-button:hover {
```

Comment:
- [ ] Dear @anbehindY. You have done an incredible job on the design. But for your buttons please follow the design guideline and use the appropriate type. 
### Figma button interactions
![image](https://user-images.githubusercontent.com/69077061/221257519-6dec3717-99ee-41ef-8218-3a369e58c866.png)